### PR TITLE
Add settings-component to Storybook

### PR DESCRIPTION
**Pull request by EtDu**

### CODE REVIEW

**Review by brad-decker — DISMISSED** [review]

**Review by brad-decker — DISMISSED** [review]

File: ui/pages/settings/settings.stories.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  [CONTACT_EDIT_ROUTE]: 'editContact',
+  [CONTACT_LIST_ROUTE]: 'contacts',
+  [CONTACT_VIEW_ROUTE]: 'viewContact',
+  [NETWORKS_ROUTE]: 'networks',
+  [NETWORKS_FORM_ROUTE]: 'networks',
+  [SECURITY_ROUTE]: 'securityAndPrivacy',
+};
+
+global.platform = {
+  getVersion: () => 'V3.14.159',
```
Comment (adonesky1) [inline ui/pages/settings/settings.stories.js:48]: Just curious why is this necessary?

Comment (EtDu) [reply ui/pages/settings/settings.stories.js]: I don't even remember lol

Comment (EtDu) [reply ui/pages/settings/settings.stories.js]: removed

**Comment by adonesky1** [comment]
@EtDu testing this out, I see the background calls in the actions console (though the arguments seem to all be empty arrays)  but not seeing the UI reflect the actions, like the radio buttons don't respond. Should we want more demonstrable actions in the storybook?

**Comment by EtDu** [comment]
Oh interesting, they are broken. Yeah I'll make sure they work so it's more of a complete component

**Review by darkwing — APPROVED** [review]

**Review by darkwing — DISMISSED** [review]

**Review by georgewrmarshall — COMMENTED** [review]
Getting `TypeError: Cannot read properties of undefined (reading 'getVersion')` when clicking on About in Settings menu

Screen Shot 2021-10-24 at 6 59 05 PM
Screen Shot 2021-10-24 at 6 59 17 PM

**Review by brad-decker — APPROVED** [review]

**Review by georgewrmarshall — APPROVED** [review]
lgtm
